2-mandatory/3-financial-times.js

```diff
@@ -24,14 +34,41 @@ function titleWithFewestWords(allArticleTitles) {
 */
 function headlinesWithNumbers(allArticleTitles) {
     // TODO
+    const arr = allArticleTitles;
+    let newArt = [];
+
+    const numbers = "1234567890";
+    for (const sentence of arr) {
+        let target = "";
+        for (const num of numbers) {
+            if (sentence.includes(num)) {
+                target = sentence;
```

Comment:
To make this slightly more efficient, you could break out of the inner loop when you've assigned to target - there's no need to check for any more numbers at this point. Have a look at https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/break